Source/items.cpp

```diff
@@ -1093,45 +1090,58 @@ void SaveItemPower(ItemStruct &item, const ItemPower &power)
 	case IPL_ACUNDEAD:
 		item._iDamAcFlags |= ISPLHF_ACUNDEAD;
 		break;
-	case IPL_MANATOLIFE:
-		r2 = ((Players[MyPlayerId]._pMaxManaBase >> 6) * 50 / 100);
-		item._iPLMana -= (r2 << 6);
-		item._iPLHP += (r2 << 6);
-		break;
-	case IPL_LIFETOMANA:
-		r2 = ((Players[MyPlayerId]._pMaxHPBase >> 6) * 40 / 100);
-		item._iPLHP -= (r2 << 6);
-		item._iPLMana += (r2 << 6);
-		break;
+	case IPL_MANATOLIFE: {
+		int fraction = ((Players[MyPlayerId]._pMaxManaBase >> 6) * 50 / 100) << 6;

```

Comment:
`fraction` doesn't really describe the purpose of this value, but I can't think of a more appropriate name 😅

Reply:
Maybe `portion`?

Reply:
Anything's better then `r2` :P The number isn't even Random :D